src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -2229,16 +2229,16 @@ private EvmExceptionType InstructionCall<TTracingInstructions, TTracingRefunds>(
         }
 
         ExecutionType executionType = GetCallExecutionType(instruction, env.IsPostMerge());
-        returnData = new EvmState(
+        returnData = EvmState.RentFrame(
```

Comment:
This and below `RentFrame` is never returned?

Reply:
Gets returned (Disposed) further up stack; otherwise we'd never be returning the Stacks to the pool

Reply:
The EvmState "Frames" are returned in `VirtualMachine.Run` as it unwinds the calls (pops them off `_stateStack`)